web/views/email/style/bulkban.html.tmpl

```diff
@@ -0,0 +1,29 @@
+{{ template "email/greeting.html" . }}
+
+{{ template "email/noticeaction.html" . }}
+
+<p>
+	Some of your styles have been removed from our platform for the following reason:<br>
+	{{ .Log.Reason }}
+</p>
+
+<p>
+	Styles that were removed:
+	<ul>
+	{{ range .Styles -}}
+		<li>
+			{{ .Name }}
+		</li>
+	{{ end }}
+	</ul>
```

Comment:
Why is this list inside of a `p` tag?

Reply:
To somewhat separate it from other parts and for the consistency as now all the parts are kept inside such tags.  
I don't really know HTML and how to write in it, nor have I tested how this email would look like rendered yet.  
Wrongness of these `p`s was already discussed a bit when we were moving these email templates to files but I still don't know how that should be done.